packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.jsx

```diff
@@ -241,7 +241,7 @@ export const IdvDocSubmitOnSignup = ({ citizen_data, has_previous, onPrevious, o
                                                                                     'Choose the document type'
                                                                                 )}
                                                                                 list_items={document_list}
-                                                                                value={values.document_type.text ?? ''}
```

Comment:
This will cause a TrackJs issue to happen again, can you find the root cause of it.
Also it is worth checking for the same issue on `idv-document-submit.jsx`

Reply:
Let me check on it 👌